src/wallet/rpc/wallet.cpp

```diff
@@ -220,7 +220,7 @@ static RPCHelpMan loadwallet()
                 "\nNote that all wallet command-line options used when starting bitcoind will be"
                 "\napplied to the new wallet.\n",
                 {
-                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The wallet directory or .dat file."},
+                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The path to the directory of the wallet to be loaded, either absolute or relative to the \"wallets\" directory. The \"wallets\" directory is set by the -walletdir option and defaults to the \"wallets\" folder within the data directory."},
```

Comment:
```suggestion
                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The path to the directory of the wallet to be loaded, either relative to the \"wallets\" directory or absolute. The \"wallets\" directory is set by the -walletdir option and defaults to the \"wallets\" folder within the data directory."},
```

The relative path seems more user-friendly, as unloading a wallet that was loaded with an absolute path might require something like

`$ ./build/src/bitcoin-cli -rpcwallet="~/Library/Application\ Support/Bitcoin/wallets/<walletname>" unloadwallet`

on macOS, or

`$ ./build/src/bitcoin-cli -rpcwallet="~/.bitcoin/wallets/<walletname>" unloadwallet`

on linux

whereas the relative path would be just

`$ ./build/src/bitcoin-cli -rpcwallet="<walletname>" unloadwallet`